pkg/controller/che/che_controller.go

```diff
@@ -352,8 +353,13 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 			users := &userv1.UserList{}
 			listOptions := &client.ListOptions{}
 			if err := r.nonCachedClient.List(context.TODO(), listOptions, users); err != nil {
+				getUsersErrMsg := failedUnableToGetOpenshiftUsers + " Cause: " + err.Error()
+				if err := r.SetStatusDetails(instance, request, failedNoOpenshiftUserReason, getUsersErrMsg, ""); err != nil {
+					return reconcile.Result{}, err
+				}
```

Comment:
Besides updating CR with the message it is needed to log the err.

Reply:
otherwise we will have just messages with log level info
```
time="2020-03-25T10:18:03Z" level=info msg="Updating eclipse-che CR with status: Message: Unable to get users on the openshift. Cause: users.user.openshift.io is forbidden: User "system:serviceaccount:che4:che-operator" cannot list users.u
time="2020-03-25T10:18:03Z" level=info msg="Custom resource eclipse-che updated".
```